gap/digraph.gd

```diff
@@ -14,6 +14,7 @@ DeclareCategory("IsDigraphWithAdjacencyFunction", IsDigraph);
 DeclareCategory("IsCayleyDigraph", IsDigraph);
 DeclareCategory("IsImmutableDigraph", IsDigraph);
 DeclareSynonym("IsMutableDigraph", IsDigraph and IsMutable);
+DeclareFilter("IsNotDefaultEdgeLabelled", IsDigraph);
```

Comment:
```suggestion

```
I think you meant to delete this line

Reply:
Thanks @MTWhyte! Just did this!

Reply:
Thanks @marinaanagno! Let's see if @james-d-mitchell has anything to say before merging.